src/HotChocolate/ElasticSearch/test/Data.ElasticSearch.Tests/__snapshots__/StringPrefixTests.ElasticSearch_AndField.snap

```diff
@@ -0,0 +1,12 @@
+﻿{
```

Comment:
Could it be that the query snapshot is missing for this one?

Reply:
It should be found in `__snapshots__/StringPrefixTests.ElasticSearch_AndField_query.snap` and it is checked to the repository. Could you double check that you don't have such file?